RentACar/RentACar/Repositories/ClientCRUD.cs

```diff
@@ -0,0 +1,54 @@
+﻿using RentACar.Models;
+using RentACar.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RentACar.Repositories
+{
+    public class ClientCRUD : Repository<Client>, IClientCRUD
+    {
+
+        private readonly string _jsonFile;
+
+        public ClientCRUD(FilePathsStorage config) : base(config.Client)
+        {
+            _jsonFile = config.Client;
+        }
+
+        public override Client Create(Client client)
+        {
+            var clients = base.GetAll();
```

Comment:
Should i call the method of this class? I called the base method because i think that i don't need an ordered list here

Reply:
in this case, it's technically the same since you already have the list loaded in memory, nevertheless I would suggest you query directly the list here, you are not forced you go through the base methods all the time, they are just there to help you in case you need them but nothing else.

Reply:
Okey! thanks for that! i will make some changes!